### DISCUSSION TITLE
Tailwind example to style everything users typically expect: links, "h1, h2, ..." "ul, li,..", etc.

### DISCUSSION THREAD

**Discussion by jcalfee** [tailwindlabs/tailwindcss#14301] [Ideas]
Could you please give a bit of guidance on how to get up and going quickly where all the default HTML have some sort of style?

For example, I start with a tailwind sold-start app and install a markdown plugin then I'm baffled as to why the Markdown seams to be ignored.  The plugin did its job, but I see that tailwind seams to reset all the styles where there is no class?   This is fine for a large project, but that is not always the case.  Here I'm just making small apps and reports quickly.   Can we please have a quick blurb or example on how to quickly plugin in all the expected classes and apply them to elements without a style?  I think this would greatly help many get up and going with your awesome product.

Thank you..

**Comment by wongjn** [comment]
https://github.com/tailwindlabs/tailwindcss-typography
